recipes/uwsgi/build.sh

```diff
@@ -0,0 +1,4 @@
+UWSGI_PROFILE="$RECIPE_DIR/uwsgi_config.ini" \
+UWSGI_INCLUDES="$PREFIX/include,$PREFIX/include/openssl" \
+    LDFLAGS="-L$PREFIX/lib" \
+    $PYTHON setup.py install
```

Comment:
As this uses `setuptools`, we need to add these arguments here ` --single-version-externally-managed --record record.txt`.